core/dataflow/src/main/resources/udf_operator.py

```diff
@@ -0,0 +1,112 @@
+from abc import ABC
```

Comment:
what does import ABC do?

Reply:
This indicates a class is an Abstract Base Class. It was suggested by my IDE when I haven't implemented all the derived methods in `MapOperator`. But since for unimplemented classes I simply use `pass`, this `ABC` can be deleted.

Reply:
Done.